src/applications/discharge-wizard/tests/v2/DischargeMonth.unit.spec.js

```diff
@@ -12,11 +11,7 @@ const mockStoreStandard = {
   getState: () => ({
     dischargeUpgradeWizard: {
       duwForm: {
-        form: {
-          SERVICE_BRANCH: null,
-          DISCHARGE_YEAR: null,
-          DISCHARGE_MONTH: null,
-        },
+        form: {},
```

Comment:
Curious - don't the display conditions require Discharge Year to be answered first?

Reply:
Oh ok gotcha, thanks!